model/event/subscribers.go

```diff
@@ -0,0 +1,68 @@
+package event
+
+import (
+	"net/url"
+
+	"github.com/mongodb/anser/bsonutil"
+	"github.com/pkg/errors"
+	"gopkg.in/mgo.v2/bson"
+)
+
+const (
+	ResourceTypeNotification = "notification"
```

Comment:
nit: can this be private?

Reply:
That shouldn't event be here, removed it.